kalite/distributed/management/commands/screenshots.py

```diff
@@ -0,0 +1,338 @@
+import errno
+import json
+import os
+import glob
+import re
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.management import call_command
+from django.core.management.base import BaseCommand, CommandError
+from django.core.urlresolvers import reverse
+from django.utils.six import StringIO
+
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+
+from fle_utils.general import ensure_dir
+
+from kalite.testing.base import KALiteBrowserTestCase
+from kalite.testing.mixins import FacilityMixins, BrowserActionMixins
+
+from optparse import make_option
+
+try:
+    from local_settings import *
+    import local_settings
+except ImportError:
+    local_settings = object()
```

Comment:
I'm a little confused as to why this is in a `try` block... under what conditions could it throw an `ImportError`?